cylc/flow/task_events_mgr.py

```diff
@@ -1510,7 +1510,6 @@ def _reset_job_timers(self, itask):
                     'execution time limit polling intervals')
                 if not time_limit_delays:
                     time_limit_delays = [60, 120, 420]
-                timeout = time_limit + sum(time_limit_delays)
```

Comment:
Using the `execution time limit` as a default for the `execution timeout` makes sense I think. Allowing this to overwrite the configured value is a bug.

How about:

```python
if not timeout:
    timeout = time_limit + P1M
```

?

Reply:
I thought about this. However, what is the purpose of the timeout? If the task has reached it's time limit then you should expect to receive a message that the task has completed so the timeout shouldn't happen (unless you're using polling in which case the timeout is fairly meaningless).
Now that we have `execution time limit` I think the main use case for `execution timeout` is to set a time shorter than the time limit - e.g. this task is taking longer than normal.

Reply:
These timeout events could be passed on to handler scripts and ingested into monitoring systems or whatnot, but yes, I wouldn't *expect* the timeout event to occur at all for a default configuration.

Reply:
I don't think setting the timeout to a fixed +PT1M offset makes sense. If we really think there is a requirement for this then I'd propose we change `execution timeout` to accept offsets to the `execution time limit` as well as absolute periods (but that's beyond the scope of this PR).